website/docs/discover-podman-desktop.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 2
+title: Discover Podman Desktop
+description: Discovering the Podman Desktop application
+keywords: [podman desktop, podman, Discover features]
+tags: [podman-desktop, discover-podman-desktop, features]
+---
+
+# Discover Podman Desktop
+
+Podman Desktop is a community-driven tool that is an alternative to Podman and Docker tools. You can perform a wide range of basic and advanced tasks. From creating a container or pod to setting up a kubernetes cluster to running an AI application to integrating your local tools using extensions, Podman Desktop offers all these features to ease your daily development tasks. But, what makes Podman Desktop different from other similar community-driven tools?
+
+The key differentiator is its intuitive UI interface that allows you to visualize your development environment. Be it the number of pods or Kubernetes clusters running or the number of deployments available or the number of services exposed, you can visualize your dev environment with a click. In addition, you have the option of running this tool on three different operating systems; macOS, Windows, and Linux. Podman Desktop runs your workloads on a Podman engine and therefore, provides you a Podman-native capabilities to interact with your applications. But, this does not mean you cannot run your Docker applications. You can use the Docker compatibility UI feature for a smooth transition to Podman Desktop.
+
+Let’s quickly explore this tool’s key functionalities.
```

Comment:
I must be honest, if I open a documentation with that much text I will not be reading it. The content is interesting, but maybe too compact as an introduction

Reply:
@axel7083, Thank you for being honest :)
If you look at the content structure in entirety, the subsequent sections discuss about key features in brief. We have not build much context there. So, having a bit of context in the beginning would help readers to relate with key functionalities of Podman Desktop described later.